source/OutfitterPanel.cpp

```diff
@@ -694,6 +697,10 @@ void OutfitterPanel::FailSell(bool toStorage) const
 	const string &verb = toStorage ? "uninstall" : "sell";
 	if(!planet || !selectedOutfit)
 		return;
+	else if(!selectedOutfit->CanSell() && !toStorage)
+		GetUI()->Push(new Dialog("You cannot " + verb + " this outfit."));
+	else if(!selectedOutfit->CanUninstall() && toStorage)
```

Comment:
An outfit that cannot be uninstalled also cannot be sold once it is installed (since selling an installed outfit automatically uninstalls it), so I guess the `toStorage` check here is not needed.

Reply:
Which line are we talking about? The check on 700, or 702?

Reply:
The check on line 702; if the outfit cannot be uninstalled, then it doesn't matter if the player tries to put it to storage or if the player tries to sell it; both cases should cause an not-allowed error-message to pop-up.